src/contrib/mlir/lowerer.cpp

```diff
@@ -771,7 +770,7 @@ namespace
     }
 
 #undef REWRITER
-/// End of pattern matchers
+    /// End of pattern matchers
     template <typename OP>
```

Comment:
We probably need a white line between comment and template so that clang format keeps the comment wo indentation